float32 cpu_usage
float32 temp
